.circleci/config.yml

```diff
@@ -1,120 +1,79 @@
 version: 2.1
 
-# add your orb below, to be used in integration tests (note: a @dev:alpha
-# release must exist; you'll need to publish manually at least once)
-# you can use scripts/publish-alpha.sh to publish a @dev:alpha release
 orbs:
-  codacy: codacy/base@dev:alpha
-  orb-tools: circleci/orb-tools@8.5.0
+  codacy: codacy/base@0.0.13
+
+references:
+  circleci_job: &circleci_job
+    docker:
+      - image: circleci/circleci-cli:0.1.5705
+    working_directory: ~/workdir
+
+jobs:
+  pack_and_validate:
+    <<: *circleci_job
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Pack orb
+          command: circleci config pack src > orb.yml 
+      - run:
+          name: Validate orb
+          command: circleci orb validate orb.yml
+      - persist_to_workspace:
+          root: ~/
+          paths:
+            - workdir/orb.yml
+
+  publish_dev:
+    <<: *circleci_job
+    steps:
+      - attach_workspace:
+          at: ~/
+      - deploy:
+          name: Publish orb as dev (deleted after 90 days)
+          command: circleci orb publish orb.yml codacy/base@dev:$(cat .version) --token $CIRCLE_TOKEN
+
+  publish_prod:
+    <<: *circleci_job
+    steps:
+      - attach_workspace:
+          at: ~/
+      - deploy:
+          name: Publish final orb
+          command: circleci orb publish orb.yml codacy/base@$(cat .version) --token $CIRCLE_TOKEN
 
 workflows:
-  # this `lint-pack_validate_publish-dev` workflow will run on any commit
-  lint-pack_validate_publish-dev:
+
+  pack_validate_publish:
     jobs:
-      # lint your destructured orb YAML files
-      - orb-tools/lint
```

Comment:
argh I think this is the check that circle ci is waiting for, can you please re-enable it? cc @h314to